SpatialGDK/Documentation/content/glossary.md

```diff
@@ -20,7 +20,7 @@ Note that this SpatialOS documentation assumes you are developing a SpatialOS ga
 ## GDK for Unreal terms
 
 ### Actor groups
-To facilitate offloading, we've created the concept of Actor groups to help you configure which Actor types a given server-worker type will have authority over. In the Unreal Editor, you can create Actor groups, assign Actor classes to a group, and then assign each group to a server-worker type.
+To facilitate [offloading](#offloading), we've created the concept of Actor groups to help you configure which Actor types a given server-worker type has [authority](#authority) over. In the Unreal Editor, you can create Actor groups, assign Actor classes to a group, and then assign each group to a server-worker type.
```

Comment:
The register here is different to the glossary and the rest of the docs. It's chatty, rather than factual. "We've created the concept of". (Reads like it's lifted from the blog?)

Suggest we make this a statement... as we know this is a new term we have created. We are in the glossary.
```suggestion
Actor groups facilitate multiserver functionality through [offloading](#offloading);  you set them up to configure which Actor types instances of a [server-worker type](#worker-types-and-instances) has [authority](#authority) over. In the Unreal Editor, you can create Actor groups, assign Actor classes to a group, and then assign each group to a server-worker type via the SpatialOS Runtime Settings panel.
```

Reply:
Yup, this was a non-tech-writered addition. Will update.